templates/bake/Plugin/tests/bootstrap.php.twig

```diff
@@ -55,3 +55,18 @@ if (file_exists($root . '/config/bootstrap.php')) {
 
     return;
 }
+
+/**
+ * Load schema from a SQL dump file.
+ *
+ * If your plugin does not use database fixtures you can
+ * safely delete this.
+ *
+ * If you want to support multiple databases, consider
+ * using migrations to provide schema for your plugin,
+ * and using \Migrations\TestSuite\Migrator to load schema.
```

Comment:
I contemplated adding an option to include database fixture setup which would include:

* Adding migrations as dev-dep
* Using the migrator instead.

If folks prefer this approach I can do that as a follow up.

Reply:
I'm not so sure about the name `Migrator` after seeing it in docs.

Reply:
We can still rename it if you have a more appropriate name in mind.